src/main/webapp/js/user-page-loader.js

```diff
@@ -30,21 +30,23 @@ function setPageTitle() {
 }
 
 /**
- * Shows the message form if the user is logged in and viewing their own page.
+ * Shows the message form if the user is logged in, even if they
+ * aren't viewing their own page.
  */
-function showMessageFormIfViewingSelf() {
-  fetch('/login-status')
-      .then((response) => {
-        return response.json();
-      })
-      .then((loginStatus) => {
-        if (loginStatus.isLoggedIn &&
-            loginStatus.username == parameterUsername) {
-          const messageForm = document.getElementById('message-form');
-          messageForm.classList.remove('hidden');
-        }
-      });
-}
+function showMessageFormIfLoggedIn() {
+	  fetch('/login-status')
+	      .then((response) => {
+	        return response.json();
+	      })
+	      .then((loginStatus) => {
+	        if (loginStatus.isLoggedIn) {
+	          const messageForm = document.getElementById('message-form');
+	          messageForm.action = '/messages?recipient=' + parameterUsername;
```

Comment:
I feel like this is adding messages whose recipient will always be the logged in user. Maybe there is a way to modify the form so that it can include a recipient as well?

Reply:
The guide (step 5-6) have this code posted, where everyone can is the "recipient" so anybody can see your code, I'm not exactly sure how that would work?
I'm currently just messing with how to message one person only, but I would appreciate it if you could walk me through how this would be for everyone to see (unless I'm interpreting it wrong). Thank you!